perf_test/KokkosKernels_perf_test_utilities.hpp

```diff
@@ -0,0 +1,146 @@
+//@HEADER
+// ************************************************************************
+//
+//                        Kokkos v. 4.0
+//       Copyright (2022) National Technology & Engineering
+//               Solutions of Sandia, LLC (NTESS).
+//
+// Under the terms of Contract DE-NA0003525 with NTESS,
+// the U.S. Government retains certain rights in this software.
+//
+// Part of Kokkos, under the Apache License v2.0 with LLVM Exceptions.
+// See https://kokkos.org/LICENSE for license information.
+// SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
+//
+//@HEADER
+//
+// Created by Berger-Vergiat, Luc on 2/6/23.
+//
+
+#ifndef KOKKOSKERNELS_PERF_TEST_UTILITIES_HPP
+#define KOKKOSKERNELS_PERF_TEST_UTILITIES_HPP
+
+// Namepsace that defines common utilities
+// for performance tests
+namespace perf_test {
+
+struct CommonInputParams {
+  int use_cuda    = 0;
+  int use_hip     = 0;
+  int use_sycl    = 0;
+  int use_openmp  = 0;
+  int use_threads = 0;
+};
+
+std::string list_common_options() {
+  std::ostringstream common_options;
+  common_options
+      << "\t[Required] BACKEND:\n"
+      << "\t\t'--threads [numThreads]' |\n"
+      << "\t\t'--openmp [numThreads]' |\n"
+      << "\t\t'--cuda [deviceIndex]' |\n"
+      << "\t\t'--hip [deviceIndex]' |\n"
+      << "\t\t'--sycl [deviceIndex]'\n\n"
+      << "\tIf no parallel backend is requested, Serial will be used "
+         "(if enabled)\n\n";
```

Comment:
Consider adding '--' to the end to denote the beginning of 'application' options.